checkov/terraform/checks/resource/aws/LambdaXrayEnabled.py

```diff
@@ -0,0 +1,18 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+class LambdaXrayEnabled(BaseResourceCheck):
+    def __init__(self):
+        name = "X-ray tracing is enabled for Lambda"
+        id = "CKV_AWS_50"
+        supported_resources = ['aws_lambda_function']
+        categories = [CheckCategories.GENERAL_SECURITY]
```

Comment:
maybe should be under the logging category? WDYT?

Reply:
It's your call. It didnt feel like it fitted anywhere and there's a bunch of xray tickets, which is more interactive than logging. ill change it to logging?